### PR TITLE
Added in views and controllers for friends and events

### PR DESCRIPTION
**Pull request by urrik98**

### CODE REVIEW

File: client/app/controllers/profile.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+angular.module('app.profile',['app.services'])
+
+.controller('ProfileCtrl', ['$scope','$state', '$window', function ($scope, $state, $window) {
+  $scope.user = {};
+  $scope.user.username = $window.sessionStorage.getItem('wefeast.user.username');
+  $scope.user.firstname = $window.sessionStorage.getItem('wefeast.user.first');
+  $scope.user.lastname = $window.sessionStorage.getItem('wefeast.user.last');
+  $scope.user.location = $window.sessionStorage.getItem('wefeast.user.location');
```
Comment (dianajohnson13) [inline client/app/controllers/profile.js]: why is the word 'location' highlighted here?

File: client/app/controllers/auth.js [hunk 2]
```diff
@@ -10,26 +16,32 @@ angular.module('app.auth', ['app.services'])
...
+          $scope.reset();
+          $state.go('main.signin');
         }
       })
       .catch(function (error) {
         console.error(error);
       });
   };
+
   $scope.advanceToFoodPrefs = function() {
```
Comment (n-9891-b) [inline client/app/controllers/auth.js]: Separate parenthesis from 'function'

File: client/app/controllers/auth.js [hunk 3]
```diff
@@ -10,26 +16,32 @@ angular.module('app.auth', ['app.services'])
...
+  };
+
   $scope.signup = function () {
-    Process.profile($scope.user)
+    var data = Profile.processData($scope.user);
+    console.log(data)
+    Auth.signup(data)
       .then(function (token) {
-
+        console.log("token received ",token)
```
Comment (n-9891-b) [inline client/app/controllers/auth.js]: Stray console.log

File: client/app/controllers/editprofile.js [hunk 4]
```diff
@@ -1,13 +1,18 @@
 'use strict';
 angular.module('app.editprofile', ['app.services'])
 
-.controller('EditProfileCtrl', ['$state','$scope', 'Profile', 'Auth', function($state, $scope, Profile, Auth) {
+.controller('EditProfileCtrl', ['$state','$scope', 'Profile', 'Auth', '$window', function($state, $scope, Profile, Auth, $window) {
 
-  $scope.user = Auth.userData;
-  console.log($scope.user)
```
Comment (n-9891-b) [inline client/app/controllers/editprofile.js]: Stray console.log

File: client/app/controllers/createprofile.js [hunk 5]
```diff
@@ -6,11 +6,11 @@ angular.module('app.createprofile', ['app.services'])
   $scope.user = $stateParams.user;
 
   $scope.activate = function(formData) {
```
Comment (n-9891-b) [inline client/app/controllers/createprofile.js]: Space needed between function and parenthesis

File: client/app/controllers/restaurantResults.js [hunk 6]
```diff
@@ -4,7 +4,11 @@ angular.module('app.restaurantresults', [])
...
   }
-  
+  $scope.databinLeft = restaurantFactory.databinLeft;
+  $scope.databinRight = restaurantFactory.databinRight;
+
+  console.log($scope.databinLeft);
+  console.log($scope.databinRight);
 
   $scope.logger = function() {
     console.log('hit');
```
Comment (n-9891-b) [inline client/app/controllers/restaurantResults.js]: Stray console.log.

**Pull request merged**